nuxeo-core/nuxeo-core-schema/src/main/java/org/nuxeo/ecm/core/schema/PropertyDescriptor.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Kevin Leturc <kleturc@nuxeo.com>
+ */
+package org.nuxeo.ecm.core.schema;
+
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.nuxeo.common.xmap.annotation.XNode;
+import org.nuxeo.common.xmap.annotation.XObject;
+import org.nuxeo.runtime.model.Descriptor;
+
+/**
+ * Descriptor representing a Nuxeo Property.
+ * <p>
+ * It maps the xml below:
+ * <pre>
+ * {@code <property schema="SCHEMA" name="NAME" secured="true" />}
+ * </pre>
+ *
+ * @since 11.1
+ */
+@XObject("property")
+public class PropertyDescriptor implements Descriptor {
+
+    @XNode("@schema")
+    protected String schema;
+
+    @XNode("@name")
+    protected String name;
+
+    @XNode("@secured")
+    public Boolean secured;
```

Comment:
`boolean`?

Reply:
In fact, after the whole work I will merge `PropertyDeprecationDescriptor` into this one, so I need the nullity of the field. (To avoid churn 😛)